messaging.rst

```diff
@@ -276,6 +276,148 @@ This should match `the Secret message in encoding/messages file <https://github.
+- The new total_withdraw value should not cause an underflow or overflow.
+- The message should be sent by one of the channel participants.
+- The :term:`nonce` is increased by ``1`` with respect to the previous :term:`nonce`
+- The ``signature`` must be from the :term:`sender` of the request.
+
+
+Fields
+^^^^^^
+
++-------------------------------+---------------+----------------------------------------------------------------+
+| Field Name                    | Field Type    |  Description                                                   |
++===============================+===============+================================================================+
+|  cmdid                        | one byte      | Value 16 (indicating ``Withdraw Confirmation``)                |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  chain identifier             | uint256       | See :ref:`balance-proof-offchain                               |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  channel identifier           | uint256       | See :ref:`balance-proof-offchain`                              |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  token network address        | address       | See :ref:`balance-proof-offchain`                              |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  message identifier           | uint64        | An ID used in ``Delivered`` and ``Processed`` acknowledgements |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  participant                  | address       | The address of the withdraw requesting node                    |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  total_withdraw               | uint256       | The new monotonic ``total_withdraw`` value                     |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  expiration                   | uint256       | The block number at which withdraw request is no longer        |
+|                               |               | usable on-chain.                                               |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  nonce                        | uint64        | See :ref:`balance-proof-offchain`                              |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  signature                    | bytes         | Elliptic Curve 256k1 signature                                 |
```

Comment:
Also, more information on what is being signed would help I think

Reply:
Done